### PR TITLE
Typescript linting fixes

### PR DESCRIPTION
**Pull request by tomlarkworthy**
A ton of typescript fixes in tests and core code

requires https://github.com/markusahlstrand/cloudworker-proxy/pull/131 to be merged first

fixes https://github.com/markusahlstrand/cloudworker-proxy/issues/121

found a few bugs along the way

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = {
     ico: 'image/microsoft.vnd.icon',
     jpeg: 'image/jpeg',
     js: 'application/javascript',
-    json: 'appliation/json',
+    json: 'application/json',
```
Comment (tomlarkworthy) [inline src/constants.ts:16]: bug fix

Comment (markusahlstrand) [reply src/constants.ts:16]: Oops.. good find

**Review by markusahlstrand — APPROVED** [review]
Really nice. Now it feels like a moderna project again :)

**Comment by tomlarkworthy** [comment]
as of this second it merges but not sure if it will stay that way after the Ts one is merged.

**Comment by markusahlstrand** [comment]
Oops, looks like the other PR i merged caused some issues...

**Comment by tomlarkworthy** [comment]
ok merge problems fixed.

File: test/handlers/cors.test.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-const { expect } = require('chai');
-const CorsHandler = require('../../src/handlers/cors');
-const helpers = require('../helpers');
+import { expect } from 'chai';
```
Comment (markusahlstrand) [inline test/handlers/cors.test.ts:1]: Maybe this isn't needed any longer as when we use jest, right?

**Pull request merged**